Task01: Pages Created

lvk1nhas opened the pull request:
* Partnership
* Voluntariado

lvk1nhas committed:
Task01: Pages Created
* Partnership
* Voluntariado

edualb commented on website/src/components/Partnership.vue:
```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+const currentYear = new Date().getFullYear();
```
I think that variable `currentYear` is not used. You can remove it from here. 

Same as https://github.com/esf-rio/esf-rio.github.io/pull/1/files#diff-343d6070d6edfc53ad7077352d5a6a971048874908dc09223ac8ee5746898739R2

edualb committed:
fixed content to fit better with bootstrap ecosystem

edualb committed:
bugfix: footer bottom

edualb commented on website/src/components/Voluntariado.vue:
```diff
@@ -0,0 +1,117 @@
+</script>
+
+<template>
+    <div class="fundo-fixo position-relative">
```
in order to keep the border rounded, I would suggest to include the class `rounded`:
https://getbootstrap.com/docs/5.3/utilities/borders/#radius

lvk1nhas replied:
Resolved

edualb commented on website/src/components/Voluntariado.vue:
```diff
@@ -0,0 +1,117 @@
+                    <p class="mb-4">Se você tem interesse em participar ativamente dos projetos ou gestão do ESF-Rio, assista o vídeo institucional do Engenheiros Sem Fronteiras Brasil e entenda um pouco como funciona nosso trabalho.</p>
+                </div>
+                <div class="mb-4 col-12 col-md-4">
+                    <video class="elementor-video" style="width: 100%; height: 100%;" controls="true" src="https://esf-rio.org/wp-content/uploads/2021/05/A-engenharia-que-transforma-vidas.mp4"  preload="metadata" controlslist="nodownload" poster="https://esf-rio.org/wp-content/uploads/2020/05/845a5473-objetivos_port.png"></video>
```
Be careful, all `src` tag in the code is using an image of the current website (that is working nowadays), however, since we are planning to turn down the current website and use that one, we need to download the file and add it into the `assets` folder and use the image from there.

lvk1nhas replied:
Resolved

edualb commented on website/src/components/Voluntariado.vue:
```diff
@@ -0,0 +1,117 @@
+    background-color: #F4F4F4;
+}
+
+.verde {
```
We have in our `assets/main.css` a class called `background-esf`, I suggest to use that one instead.

edualb committed:
fix video and image overflow on volunteer page

lvk1nhas committed:
Atualiza páginas Partnership e Voluntariado

- Modificações no layout e conteúdo das páginas Partnership e
  Voluntariado
- Ajustes no CSS para responsividade
- Correções de texto e links atualizadosAtualiza páginas Partnership e
    Voluntariado

lvk1nhas committed:
Atualizado Nome Voluntaria

edualb commented on website/index.html:
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <link rel="icon" href="/esf-favicon.ico">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
```
testing comment

lvk1nhas replied:
testing

lvk1nhas committed:
Assets

lvk1nhas committed:
Refactored code based on PR feedback:

- Removed unused variable `currentYear`
- Added `rounded` class to preserve border styling
- Moved external images to local assets folder and updated `src` paths

lvk1nhas committed:
Removed images and fixed padding

lvk1nhas committed:
Added git LFS: jpg, png, mp4
